package.json

```diff
@@ -33,6 +33,7 @@
     "vue-native-websocket": "^2.0.15",
     "vue-route": "^1.5.1",
     "vue-router": "^4.4.5",
+    "vue-turnstile": "^1.0.11",
```

Comment:
Consider using https://github.com/Astrian/cfturnstile-vue3
vue-turnstile is designed for Vue 2.

Reply:
https://github.com/ruigomeseu/vue-turnstile

It's designed for Vue 3 instead of Vue 2, if you have any questions about the Options API, please refer to: https://cn.vuejs.org/guide/introduction.html#options-api